lib/modules/@apostrophecms/storybook/index.js

```diff
@@ -3,18 +3,48 @@ const fs = require('fs-extra');
 const IMPORTS_PLACEHOLDER = /(\/\/ IMPORTS)/g;
 const COMPONENTS_PLACEHOLDER = /(\/\/ COMPONENTS)/g;
 const STORIES_PLACEHOLDER = /"STORIES"/g;
+const childProcess = require('child_process');
 
 module.exports = {
   init(self, options) {
-    self.addTask('build', 'update the data/temp/.storybook folder', self.build);
-    self.addTask('run', 'start storybook', self.run);
+    self.addTask('build', 'pass a folder name to build a static storybook', self.build);
+    self.addTask('run', 'start storybook dev server', self.run);
+    self.addTask('deploy', 'deploy static storybook to github pages of your project repo', self.deploy);
   },
   methods(self, options) {
     return {
       async build(apos, argv) {
+        const target = require('path').resolve(process.cwd(), argv._[1]);
+        if (!target) {
+          throw 'Must specify target directory as first argument.';
+        }
+        await self.preBuild(argv);
+        const buildDir = self.getBuildDir();
+        process.env.APOS_ROOT = self.apos.rootDir;
```

Comment:
`buildDir` and `APOS_ROOT` seem to be redefined in the `build, run, deploy` and `prebuild`. Should these be bumped up so they don't need to be redefined?